net/client.go

```diff
@@ -355,8 +355,8 @@ func (s *server) pushRecord(ctx context.Context, id thread.ID, lid peer.ID, rec
 	}
 
 	// Finally, publish to the thread's topic
-	if err = s.publish(id, req); err != nil {
-		log.Error(err)
+	if err = s.topics.Publish(ctx, id, req); err != nil {
+		return err
```

Comment:
Flashing thought: Makes sense to relax if pubsub pushing fails? Maybe log and not err?

Reply:
Yep, good call.